docs/source/tutorial/interfaces.rst

```diff
@@ -404,14 +402,14 @@ case back as follows:
 The effect of this version of ``readNumbers`` is identical to the first (in
 fact, it is syntactic sugar for it and directly translated back into that form).
 The first part of each statement (``Just x_ok <-`` and ``Just y_ok <-``) gives
-the preferred binding - if this matches, execution will continue with the rest
+the preferred binding — if this matches, execution will continue with the rest
```

Comment:
This should possibly be a semicolon or simply a full stop? In any case, it should not be an em-dash, which are used to indicate an interruption of speech in British English.